examples/galaxies/plot_ellipticity.py

```diff
@@ -0,0 +1,123 @@
+"""
+Galaxy Ellipticity Distributions
+================================
+
+This example demonstrate how to sample ellipticity distributions
+in SkyPy.
+
+"""
+
+
+# %%
+# 3D Ellipticity Distribution
+# ---------------------------
+#
+# In Ryden 2004 [1]_, the ellipticity is sampled by randomly projecting
+# a 3D ellipsoid with principal axes :math:`A > B > C`.
+#
+# The distribution of the axis ratio :math:`\gamma = C/A` is a truncated
+# normal with mean :math:`\mu_\gamma` and standard deviation
+# :math:`\sigma_\gamma`.
+#
+# The distribution of :math:`\epsilon = \log(1 - B/A)` is truncated normal
+# with mean :math:`\mu` and standard deviation :math:`\sigma`.
+#
+#
+# In ``SkyPy``, we can sample 3D ellipticities using the
+# :func:`skypy.galaxies.morphology.ryden04_ellipticity()` distribution
+# described above [1].
+#
+# Validation plot with SDSS Data
+# ------------------------------
+#
+# Here we validate our function by comparing our ``SkyPy`` simulated galaxy
+# ellipticities against observational data from SDSS DR1 in Figure 1 [1].
+# You can download the data file
+# :download:`SDSS_ellipticity <../../../examples/galaxies/SDSS_ellipticity.txt>`,
+# stored as a 2D array: ellipticity_X, ellipticity_T.
+#
+
+import numpy as np
+
+# Load SDSS data from Fig. 1 in Ryden 2004
+eX, eT = np.loadtxt('SDSS_ellipticity.txt', unpack=True)
+Ngal = len(eX)
+e = np.hypot(eX, eT)
+q_amSDSS = np.sqrt((1 - e)/(1 + e))
+
+
+# %%
+#
+# In this example, we generate 100 galaxy ellipticity samples using the
+# ``SkyPy`` function
+# :func:`skypy.galaxies.morphology.ryden04_ellipticity()` and the
+# best fit parameters
+# given in Ryden 2004 [1]:
+# :math:`\mu_\gamma =0.222`, :math:`\sigma_\gamma=0.057`, :math:`\mu =-1.85`
+# and :math:`\sigma=0.89`.
+#
+
+from skypy.galaxies.morphology import ryden04_ellipticity
+
+# Best fit parameters from Fig. 1 in Ryden 2004
+mu_gamma, sigma_gamma, mu, sigma = 0.222, 0.057, -1.85, 0.89
+
+# Binning scheme of Fig. 1
+bins = np.linspace(0, 1, 41)
+mid = 0.5 * (bins[:-1] + bins[1:])
+
+# Mean and variance of sampling
+mean = np.zeros(len(bins)-1)
+var = np.zeros(len(bins)-1)
+
+# %%
+#
+
+# Create 100 SkyPy realisations
+for i in range(100):
+    # sample ellipticity
+    e = ryden04_ellipticity(mu_gamma, sigma_gamma, mu, sigma, size=Ngal)
+    # recover axis ratio from ellipticity
+    q = (1 - e)/(1 + e)
```

Comment:
Is `q_am` not the square root of this according to equation (9) from [Ryden (2014)](https://arxiv.org/pdf/astro-ph/0310097.pdf)?

Reply:
I took this from @ntessore reproduce page. I have the same question.

Reply:
[No](https://github.com/skypyproject/skypy/blob/cea1f02d1b2cd3b689266d7ae9bca1a4cfe986a2/skypy/galaxies/morphology.py#L284). Our definition of ellipticity should always be `(a-b)/(a+b) = (1-q)/(1+q)`, there is issue #231 on exactly this matter.

Reply:
In this case the PR is fine for me after gzipping the file.